mmv1/third_party/terraform/services/appengine/resource_app_engine_flexible_app_version_test.go

```diff
@@ -53,6 +53,19 @@ resource "google_project" "my_project" {
   billing_account = "%{billing_account}"
 }
 
+resource "google_compute_network" "network" {
```

Comment:
```suggestion
resource "google_compute_network" "network" {
  project = google_project.my_project.project_id
```

+ same thing for the subnetwork, and both subnetwork+network in the next step! We're creating these in the wrong project, I didn't catch it before because we were getting an error we couldn't update the network at all.

Reply:
Done